auditors/mounts/mounts_test.go

```diff
@@ -0,0 +1,34 @@
+package mounts
+
+import (
+	"github.com/Shopify/kubeaudit/internal/override"
+	"strings"
+	"testing"
+
+	"github.com/Shopify/kubeaudit/internal/test"
+)
+
+const fixtureDir = "fixtures"
+
+func TestSensitivePathsMounted(t *testing.T) {
+	cases := []struct {
+		file           string
+		fixtureDir     string
+		expectedErrors []string
+	}{
+		{"docker-sock-mounted.yml", fixtureDir, []string{SensitivePathsMounted}},
+		{"proc-mounted.yml", fixtureDir, []string{SensitivePathsMounted}},
+		{"proc-mounted-allowed.yml", fixtureDir, []string{override.GetOverriddenResultName(SensitivePathsMounted)}},
+		{"proc-mounted-allowed-multi-containers-multi-labels.yml", fixtureDir, []string{override.GetOverriddenResultName(SensitivePathsMounted)}},
+		{"proc-mounted-allowed-multi-containers-single-label.yml", fixtureDir, []string{SensitivePathsMounted, override.GetOverriddenResultName(SensitivePathsMounted)}},
+	}
+
+	config := Config{SensitivePaths: DefaultSensitivePaths}
```

Comment:
I think we can let this fallback to the default in the config now?
```suggestion
	config := Config{}
```